### DISCUSSION TITLE
How to save a game?

### DISCUSSION THREAD

**Discussion by ldesousa** [stefanhendriks/Dune-II---The-Maker#352] [Q&A] [answered]
Hi there, congrats for this work, really sends me back in time.

I played a couple of times but I cannot figure out how to save the game/campaign to resume another day. Is such feature available? How to use it?

Thank you.

**Comment by stefanhendriks** [comment, marked as answer]
There is no way to save the game yet. It is on the list to build. But that will not be in the short future. Alternatively I am thinking of adding a mission select in the near future so you can atleast start from a certain point in the campaign(s).
